### ISSUE TITLE
Multilingual

### ISSUE THREAD

**Issue by vahidtakro** [colbyfayock/my-next-mdx#1]
Thanks for the tutorial!
how can I make it multilingual ?

**Comment by colbyfayock** [comment]
hey @vahidtakro check out this documentation for internationalization in Next.js: https://nextjs.org/docs/advanced-features/i18n-routing

You could likely couple that to get what you need

**Comment by vahidtakro** [comment]
Yes I've done that already, but still I want to translate the markdown blog, I don't know how to do that
